src/Components/ADT3DSceneBuilder/Internal/Shared/PanelForms.styles.ts

```diff
@@ -15,7 +16,7 @@ const classNames = {
     pivot: `${classPrefix}-pivot`
 };
 export const getPanelFormStyles = memoizeFunction(
-    (_theme: Theme, formHeaderHeight: number) => {
+    (_theme: Theme, formHeaderHeight: number, padContent = false) => {
```

Comment:
nit: changing the name to reflect it is a bool
```suggestion
    (_theme: Theme, formHeaderHeight: number, shouldPadContent = false) => {
```